airbyte-integrations/connector-templates/source-python-http-api/acceptance-test-config.yml.hbs

```diff
@@ -0,0 +1,29 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/source-acceptance-tests.md)
```

Comment:
not needed for this PR but the number of common files between these templates is growing. Maybe we should have a common directory `connector-templates/common` which contains all common files.

Reply:
if we will migrate to cookiecuttter we can have a single template for all types of connectors, solving this problem in such case

Reply:
#3049